bitsail-connectors/connector-kafka/src/main/java/com/bytedance/bitsail/connector/kafka/option/KafkaWriterOptions.java

```diff
@@ -24,9 +24,10 @@
 import static com.bytedance.bitsail.common.option.WriterOptions.WRITER_PREFIX;
 
 public interface KafkaWriterOptions extends WriterOptions.BaseWriterOptions {
-  ConfigOption<String> KAFKA_SERVERS =
-      key(WRITER_PREFIX + "kafka_servers")
-          .noDefaultValue(String.class);
+  ConfigOption<String> BOOTSTRAP_SERVERS =
+      key(WRITER_PREFIX + "bootstrap_servers")
+          .noDefaultValue(String.class)
+          .withAlias(WRITER_PREFIX + "kafka_servers");
```

Comment:
@liuxiaocs7 FYI, found a way to make this change

Reply:
> @liuxiaocs7 FYI, found a way to make this change

got it, thanks, `alias` is a good choice for me.